app/classes/Transvision/ShowResults.php

```diff
@@ -461,7 +504,9 @@ public static function resultsTable($search_object, $search_results, $page)
                       <a class='bug_link' target='_blank' href='{$bz_link[0]}'>
                         &lt;report a bug&gt;
                       </a>
-                      {$meta_target}
+                      {$meta_target}";
```

Comment:
You forgot to remove this part, only {$meta_target} should remain here, since we’re defining its content earlier